src/test/java/seedu/address/logic/parser/EditBuyerCommandParserTest.java

```diff
@@ -172,27 +174,12 @@ public void parse_oneFieldSpecified_success() {
         assertParseSuccess(parser, userInput, expectedCommand);
 
         // tags
-        userInput = targetIndex.getOneBased() + TAG_DESC_FRIEND;
-        descriptor = new EditPersonDescriptorBuilder().withTags(VALID_TAG_FRIEND).build();
+        userInput = targetIndex.getOneBased() + TAG_DESC_PRIORITY_HIGH;
+        descriptor = new EditPersonDescriptorBuilder().withPriority(VALID_PRIORITY_HIGH).build();
         expectedCommand = new EditBuyerCommand(targetIndex, descriptor);
         assertParseSuccess(parser, userInput, expectedCommand);
     }
 
-    @Test
```

Comment:
Is this test case not needed anymore? It checks for repeated flags apart from tags like multiple `-n AMY -n BOB`

Reply:
Yes, my bad I deleted it by accident. Thanks for catching it!